server/routes/content-fragments/index.ts

```diff
@@ -37,12 +56,20 @@ const setupCustomHeaderBasedAuthentication = (req: Request, res: Response, next:
     jwksUri: `${config.apis.hmppsAuth.url}/.well-known/jwks.json`,
   }) as GetVerificationKey
 
+  const token = req.headers['x-user-token'] as string
+
   expressjwt({
     secret: jwksIssuer,
     issuer: `${config.apis.hmppsAuth.url}/issuer`,
     algorithms: ['RS256'],
-    getToken: reqInternal => reqInternal.headers['x-user-token'] as string,
-  })(req, res, next)
+    getToken: () => token,
+    requestProperty: 'user',
```

Comment:
This is the magic source to get this auth processing working with downstream stuff ....

`requestProperty: 'user',` tells the Express JWT method to put the resultant decoded token on `req.user` 👍 

And the promise callback:
```
    res.locals.user = {
      ...res.locals.user,
      token,
    }
```
puts the token on `res.locals.user`; which is subsequently processed by the middleware call to `setUpCurrentUser` above ☝️